frontend/src/models/types/Template.ts

```diff
@@ -0,0 +1,26 @@
+import { RubricCriterion } from './RubricCriterion.ts';
+import { v4 as uuidv4 } from 'uuid';
+
+export interface Template {
```

Comment:
Similar note to the other PR, I think a whole additional data structure is unnecessary.

We could add a field to the local user data structure with ids of rubrics they want to have ready as templates